.github/workflows/commit-check.yml

```diff
@@ -0,0 +1,23 @@
+name: Commit Message Check
+
+on:
+  push:
+    branches:
+      - main # Or your default branch
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  check_commit_title:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4 # Checkout the repository
+       - name: Check Commit Message
+        uses: GsActions/commit-message-checker@v1.0.0 # Use the commit message checker action
```

Comment:
I believe policy is to use a specific commit id (SHA1) and not a tag here to mitigate plugin spoofing